### ISSUE TITLE
Keyword 'Altimetry' returns nothing.'Altimeter' does. 

### ISSUE THREAD

**Issue by Yongyao** [mudrod/mudrod#152]
@fgreg I think this has something to do with stemming. If this is the only keyword that does not work. We can just hard code this.

@lewismc Do you think your current ontology results can fix this problem, through link query expansion?

**Comment by lewismc** [comment]
@Yongyao yes this is exactly the type of thing that any implementation of the [Ontology Synonyms API](https://github.com/mudrod/mudrod/blob/master/core/src/main/java/gov/nasa/jpl/mudrod/ontology/Ontology.java#L58-L65) would address. This could be accessed internally, or externally through the [OntologyResource](https://github.com/mudrod/mudrod/blob/master/service/src/main/java/gov/nasa/jpl/mudrod/services/ontology/OntologyResource.java). Assigned it to myself.

**Comment by lewismc** [comment]
Hi @Yongyao, now that I review the process of ontology term expansion, unless there is an equialentClass attribute in the ontology(ies) resource(s), ontology will not help here. 
This has to do with stemming in Lucene, so we need to look at what kind of pre-processing is going on prior to the index construction. We should iterate on this. 

SWEET for example, maintains the concept of [Altimetry](https://github.com/ESIPFed/sweet/blob/master/2.4/reprSciMethodology.owl#L32-L34) but as you can see, it is not linked to any other concept... this is to say, that within the SWEET world, _Altimetry_  and _Altimeter_ are two different things. The former is a representation of a methodology and the latter is an instrument which acquires physical measurements of data described by the methodology.
